packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -571,6 +499,24 @@ export class GCFBootstrapper {
     };
   }
 
+  private buildLoggerBindings(
+    botRequest: BotRequest,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    requestBody: {[key: string]: any}
```

Comment:
Does express expose a better type for `body`?

Reply:
I don't believe so - it depends on the shape of the data and how it's decoded. In this case, we've JSON.parse'd the body string somehow with a filter.